packages/desktop-client/src/components/reports/graphs/CashFlowGraph.tsx

```diff
@@ -16,10 +16,17 @@ import Container from '../Container';
 import Tooltip from '../Tooltip';
 
 type CashFlowGraphProps = {
+  start: string;
+  end: string;
   graphData: { expenses; income; balances };
   isConcise: boolean;
 };
-function CashFlowGraph({ graphData, isConcise }: CashFlowGraphProps) {
+function CashFlowGraph({
+  start,
+  end,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏the component is not using `start`/`end` props. As such: we should not add them.